docs/alerts/index.md

```diff
@@ -7,105 +7,54 @@ frameborder="0" allow="autoplay; encrypted-media"
 
-For example, you might want to be notified when available disk space reaches a certain threshold - *metrics alert*, or when the number of logs with `severity: warning` gets too high - *logs alert*, or when your users start experiencing high response time - *experience alert*.
+Alerts are used to notify you when one or more pre-defined conditions in your *metrics*, *logs*, *experience*, or *synthetics* data are met.
+
+For example, you might want to be notified when available disk space reaches a certain threshold – *metrics alert*, when the number of logs with `severity: warning` gets too high – *logs alert*, when your users start experiencing high response times – *experience alert*, or when a website or API monitor detects downtime or slow response times – *synthetics alert*.
 
 <!--iframe width="800" height="450" src="https://www.youtube.com/embed/WE9xAUud28o?rel=0" frameborder="0" allow="autoplay; encrypted-media" allowfullscreen></iframe-->
 
 <br/>
 
 ## Alert Types
+
 [Sematext Cloud](https://sematext.com/cloud) includes multiple **types** of alerts that integrate with [PagerDuty](https://sematext.com/docs/integration/alerts-pagerduty-integration/), [Slack](https://sematext.com/docs/integration/alerts-slack-integration/), [email](https://sematext.com/docs/alerts/alert-notifications/#email-notification-hooks), and [other 3rd party services](alert-notifications).  
 
 1. **Threshold** alerts are the classic threshold based alerts. They are triggered when a certain pre-defined threshold is met.
-1. **Heartbeat** alerts are triggered when [Sematext Cloud](https://sematext.com/cloud) stops receiving data from your server, container, application, etc.  
-1. **Anomaly** alerts are based on statistical anomaly detection. They are triggered when values suddenly change and deviate from the continously computed baseline.
+2. **Heartbeat** alerts are triggered when [Sematext Cloud](https://sematext.com/cloud) stops receiving data from your server, container, application, etc.  
+3. **Anomaly** alerts are based on statistical anomaly detection. They are triggered when values suddenly change and deviate from the continously computed baseline.
 
 ![Sematext Anomaly Alerts](../images/guide/alerts-and-events/anomaly-alerts.png)
 
 When an anomaly alert is triggered, you'll be able to observe spikes and dips within the alert details flyout upon clicking on the [alert event](https://sematext.com/docs/alerts/alert-events/). The chart also features a confidence band that illustrates the range of certainty around the expected values.
 
 ![Sematext Anomaly Alert Confidence Band](../images/guide/alerts-and-events/anomaly-alert-confidence-band.png)
 
-<br/>
-
 ## Alert Sources
+
 Alerts can operate on three different **sources** of data:  
 
-1. **Metrics** alerts work with infrastructure metrics
-1. **Logs** alerts are based on various types of logs
-1. **Experience** alerts are concerned with *real user monitoring* data - RUM
```

Comment:
Btw this 1. 1. 1. automatically turns that into a numbered list.

Reply:
GitHub Markdown is quite different from MkDocs, so I'm trying to play it safe